docs/readmes/faq/faq_magma.md

```diff
@@ -102,9 +102,9 @@ This section lists some of the commonly asked questions related to Magma operati
   - Put in the required inputs and click **Execute**.
 
 ### How can I check the services running in Orchestrator?
-  - List the running pods with `kubectl -norc8r get pods`
+  - List the running pods with `kubectl -n orc8r get pods`
   - Grab the name of orc8r-controller pods, they are in the format `orc8r-controller-xxxxxxxxxx-yyyyy`
-  - Collect the state of services on each pod: `kubectl -norc8r exec orc8r-controller-xxxxxxxxxx-yyyyy  bash -- supervisorctl status`
+  - Collect the state of services on each pod: `kubectl -n orc8r exec orc8r-controller-xxxxxxxxxx-yyyyy  bash -- supervisorctl status`
 
 ### How to verify which services are running in Orc8r-controller?
   - **Get the controller pods:** `kubectl -n magma get pods`.
```

Comment:
Can we also fix this and the one directly below -- default namespace is orc8r not Magma. Also move the flag so that it's directly after `kubectl` -- so `kubectl -n orc8r exec`